src/NHSD.BuyingCatalogue.Solutions.Contracts/Persistence/ISolutionFrameworkListResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace NHSD.BuyingCatalogue.Solutions.Contracts.Persistence
+{
+    public interface ISolutionFrameworkListResult
+    {
+        public string Id { get; }
+
+        public string FrameworkName { get; }
```

Comment:
I think `ShortName` or `FrameworkShortName` might be better here as there is another `Name` column for the full name so I think it makes sense to distinguish between the two here. It will also make it easier if we ever want to expose the name column.

Reply:
Updated Property to `ShortName`